From e6c0d487fb5e6063305b8b3187896fbfe5720ed2 Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Mon, 22 Feb 2021 18:42:43 +0200
Subject: [PATCH] setup.py: Fix numpy version

Fix numpy version to ensure bitbake will find the pip package.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7dbe583..6602c97 100755
--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@ SETUP_REQUIRES = [
     f"Cython >=0.29; python_version<'3.8'",
     f"Cython >=0.29.14; python_version>='3.8'",
 ] + [
-    f"numpy =={np_min}; python_version{py_condition}"
+    f"numpy >={np_min}; python_version{py_condition}"
     for np_min, py_condition in NUMPY_MIN_VERSIONS
 ]
 
